source/beamline/SGM/SGMBypassLadder.h

```diff
@@ -0,0 +1,32 @@
+#ifndef SGMBYPASSLADDER_H
+#define SGMBYPASSLADDER_H
+
+#include "beamline/SGM/SGMLadderControl.h"
+
+#define SGMBYPASSLADDER_SLIT_SETPOINT 1260
+#define SGMBYPASSLADDER_SLIT_MIN 0
+#define SGMBYPASSLADDER_SLIT_MAX 1260
+
+#define SGMBYPASSLADDER_PASSTHROUGH_SETPOINT 12000
+#define SGMBYPASSLADDER_PASSTHROUGH_MIN 12000
```

Comment:
The values are sort of a mash of what was presented in the specification and what was experimentally achievable. The controls work as written, but I'm not sure how well these values match up with reality. Tom is away, it would probably be a good idea to confirm these values with him.